client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -43,6 +43,9 @@ class AuthorMappingPane extends PureComponent {
 	};
 
 	getMappingDescription = ( numSourceUsers, numTargetUsers, targetTitle, sourceType ) => {
+		if ( targetTitle === 'Site Title' ) {
+			targetTitle = 'your new site';
+		}
```

Comment:
I have two issues/concerns here:
* You're assuming the current language is English, so `your new site` may not be the right string
* How does/will this work with non-English sites? Do we translate the default `Site Title` into the site creator's language?
* What happens when the site language is not the same as the current user's language? e.g. the site may have been created using `fr`, but the current user's default language is `es`.

Reply:
My first thought was to remove the site-specific language entirely. Given the complexity around translations, this seems like the best way to go.